scripts/zkai_proof_pressure_slope_table_gate.py

```
@@ -0,0 +1,642 @@
+    for artifact, (artifact_id, path) in zip(source_artifacts, SOURCE_ARTIFACTS, strict=True):
+        if artifact.get("id") != artifact_id or artifact.get("path") != path.relative_to(ROOT).as_posix():
+            raise ProofPressureSlopeTableError("source artifact identity drift")
+        if not isinstance(artifact.get("sha256"), str) or len(artifact["sha256"]) != 64:
+            raise ProofPressureSlopeTableError("source artifact digest drift")
+    rows = payload.get("rows")
+    if not isinstance(rows, list):
+        raise ProofPressureSlopeTableError("rows missing")
+    validate_rows(rows)
+    summary = payload.get("summary")
+    if not isinstance(summary, dict) or summary.get("recommended_next_gate") != NEXT_GATE:
+        raise ProofPressureSlopeTableError("summary next-gate drift")
```

Ruff (0.15.13)

[warning] Avoid specifying long messages outside the exception class

(TRY003)